net/pfSense-pkg-Avahi/files/usr/local/pkg/avahi/avahi.inc

```diff
@@ -130,3 +150,44 @@ function avahi_deinstall_command() {
 
 	unlink_if_exists(AVAHI_RCFILE);
 }
+
+function avahi_get_carp_list() {
+	$list = array();
+	$list['none'] = 'none';
+	$carplist = get_configured_vip_list("all", VIP_CARP);
+	foreach ($carplist as $vid => $vaddr) {
+		$vip = get_configured_vip($vid);
+		$list[$vid] = "$vaddr (vhid: {$vip['vhid']})";
```

Comment:
Inside double quotes, `$vaddr` should be surrounded by curly braces: `{$vaddr}`

Reply:
done